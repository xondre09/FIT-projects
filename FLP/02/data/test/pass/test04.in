S a S R
S   F  
aaaaaa
