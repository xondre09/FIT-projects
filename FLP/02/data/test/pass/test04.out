Saaaaaa
aSaaaaa
aaSaaaa
aaaSaaa
aaaaSaa
aaaaaSa
aaaaaaS 
aaaaaaF 
